en/Basic Math/Arithmetic Progressions.md

```diff
@@ -0,0 +1,44 @@
+# Arithmetic Progression
```

Comment:
```suggestion
# Arithmetic Progression

```

Reply:
This still has to be resolved.

Reply:
@Panquesito7 : I can't able to understand, What's the change here?